.gitignore

```diff
@@ -0,0 +1,6 @@
+# IDE Stuff
+.idea
```

Comment:
I would add more files. Maybe you can adopt the structure from the other packages:

https://github.com/aboutbits/pagination/blob/master/.gitignore

Reply:
I have added the required fields